dash/__init__.py

```diff
@@ -1,3 +1,7 @@
 from .dash import Dash  # noqa: F401
+from . import authentication  # noqa: F401
```

Comment:
let's leave this one out, as it isn't used anymore (`dash-auth` package is used instead)

Reply:
Fixed!